@navikt/core/css/darkside/list.darkside.css

```diff
@@ -41,45 +61,27 @@ _:future,
 /* stylelint-enable selector-type-no-unknown */
 
 .navds-list__item {
-  margin-block-end: var(--a-spacing-2);
-}
+  margin-block-end: var(--ax-spacing-2);
 
-.navds-list__item:last-child {
-  margin-block-end: 0;
-}
-
-.navds-list ul > .navds-list__item {
-  display: grid;
-  grid-template-columns: auto 1fr;
-  gap: var(--a-spacing-2);
-}
-
-.navds-list ol li::marker {
-  font-weight: var(--a-font-weight-bold);
-  color: var(--ac-list-marker-ol-color, var(--ac-list-marker-color, var(--a-icon-default)));
+  &:last-child {
+    margin-block-end: 0;
+  }
 }
 
 .navds-list__item-marker {
   width: 1.5rem;
 }
 
-.navds-list ul > li > .navds-list__item-marker {
-  display: flex;
-  align-items: center;
-  height: var(--a-font-line-height-xlarge);
-}
-
 .navds-list--small ul > li > .navds-list__item-marker {
-  height: var(--a-font-line-height-large);
+  height: var(--ax-font-line-height-large);
 }
 
 .navds-list__item-marker--bullet {
   padding-left: 0.8rem;
-  color: var(--ac-list-marker-ul-color, var(--ac-list-marker-color, var(--a-icon-default)));
 }
 
 .navds-list__item-marker--icon {
   font-size: 1.5rem;
   justify-content: center;
-  color: var(--ac-list-marker-icon-color, var(--ac-list-marker-color, var(--a-icon-default)));
+  color: var(--ax-text-default);
```

Comment:
Can this one be removed as well?

Reply:
I think in most cases we want the icon to remain "text-default", but not 100%. Will ask in Figma what the designers might think.

Reply:
Changed it to `currentColor`. In the case they provide a custom SVG without `currentColor` set like our icons this should work as a fallback 🤞

Reply:
But the selector does not target the icon itself, but the wrapper 🤔

Reply:
That's true, remove it since it had no effect on color that default `inherit` did not.